src/modules/python.rs

```diff
@@ -121,12 +121,13 @@ fn get_python_virtual_env(context: &Context) -> Option<String> {
         })
     })
 }
+
 fn get_prompt_from_venv(venv_path: &Path) -> Option<String> {
-    Ini::load_from_file(venv_path.join("pyvenv.cfg"))
+    Ini::load_from_file_noescape(venv_path.join("pyvenv.cfg"))
         .ok()?
         .general_section()
         .get("prompt")
-        .map(|prompt| String::from(prompt.trim_matches(&['(', ')'] as &[_])))
+        .map(|prompt| String::from(prompt.replace("\\", "/").trim_matches(&['(', ')'] as &[_])))
```

Comment:
This should not be necessary with `load_from_file_noescape`
```suggestion
        .map(|prompt| String::from(prompt.trim_matches(&['(', ')'] as &[_])))
```

Reply:
Yes, you're right. At first, I wanted to convert the prompt to a posix style, but that doesn't seem necessary now.